test_mae.py

```diff
@@ -0,0 +1,84 @@
+import torch
+
+from timm.optim.lion import Lion
+from torch.optim.lr_scheduler import OneCycleLR
+from torch.nn import MSELoss, LayerNorm, LazyInstanceNorm2d, Sequential
+from hydra.utils import instantiate
+
+from mae_modules import MAEEncoder, CAMAEDecoder
+from loss import FourierLoss
+from vit import sincos_positional_encoding_vit, vit_small_patch16_256
+from mae import MAE, Normalizer
+
+model_config = {
```

Comment:
shouldn't this test just load the JSON rather than redefining it here?

Reply:
the way config is defined for our torch model (through hydra) and the hugging face is slightly different. Just keeping it separate